code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -289,7 +289,10 @@ Pipelines + Other Objects -> Pipe network
 	return list()
 
 /obj/machinery/atmospherics/update_remote_sight(mob/user)

```

Comment:
SEE_TURFS vision is bad for borers as they rely on hiding beneath objects and in people to survive.